region/maridia/inner-pink.json

```diff
@@ -3895,7 +3895,28 @@
                   "requires": [
                     "canSuitlessMaridia",
                     "HiJump",
-                    "SpaceJump"
+                    {"or": [
+                      {"and": [
+                        "canSpaceJumpWaterBounce",
+                        "canWalljump"
+                      ]},
+                      "canSpaceJumpWaterEscape"
+                    ]}
+                  ],
+                  "devNote": "FIXME: With a single underwaterwalljump you can break free here, without spacejump."
+                },
+                {
+                  "name": "Suitless SpringBall Jump",
+                  "notable": false,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "HiJump",
+                    "canTrickyJump",
+                    "h_canMaxHeightSpringBallJump"
```

Comment:
This seems really challenging. Does it need a `canStationaryLateralMidAirMorph`?

Reply:
Its either a late morph springballjump, or a `canStationaryLateralMidAirMorph`.  This is easier than the cac alley one, but still hard for very hard.